admin/resources/js/components/H2Components/Button.tsx

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 
 export interface ButtonProps {
+  /** The child elements to appear within the parent element. */
   children: React.ReactNode;
```

Comment:
I should have caught this in a previous PR, but its not necessary to define children here. `React.FunctionComponent` is defined such that a component of type `React.FC<P>` will accept props of type `P & { children?: ReactNode | undefined }`